js/image/src/Webcam.svelte

```diff
@@ -85,6 +88,11 @@
 						name: "sample." + mimeType.substring(6),
 						is_example: false
 					});
+					dispatch("stop_recording", {
```

Comment:
What is the `capture` event above @pngwn @aliabid94 ? It doesn't exist in the python api afaik

Reply:
So , I think there is some confusion here. This fix works but not because we are sending the audio data back with the event. That only necessary if we wish to populate the `EventData` argument which event related metadata. However that isn't the case here, we just need the value to be updated _before_ the `stop_recording` event is dispatched. The issues here is actually a race condition/ bug in the original code due to _when_ we are dispatching the event rather than what that event contains.

`capture` is an internal event (not public) used by the Video component to set the data when it changes. That is why I was confused about this issue, the event and the setting of data are independent in almost all components (and particularly here), `capture` handles the setting of data regardless of events.

The reason this fix works is because you moved the `stop_recording` dispatch to the `onload` of the `FileReader` instance and placed it _after_ the `capture` event, which is correct and an oversight from the original PR.

We can remove the event data as it isn't needed (and would populate the `EventData` for functions that use) which would duplicate the data and is inconsistent with other events/ components.